src/Stratis.Bitcoin.Features.MemoryPool/MempoolOrphans.cs

```diff
@@ -402,11 +421,15 @@ public Task<bool> AddOrphanTx(ulong nodeId, Transaction tx)
         /// <returns>Whether erased.</returns>
         private Task<bool> EraseOrphanTx(uint256 hash)
         {
+            this.mempoolLogger.LogTrace("{0}:'{1}'", nameof(hash), hash);
             return this.MempoolLock.WriteAsync(() =>
             {
                 OrphanTx it = this.mapOrphanTransactions.TryGet(hash);
                 if (it == null)
+                {
+                    this.mempoolLogger.LogTrace("(-):false[NOTFOUND_ORPH]");
```

Comment:
`false[NOTFOUND_ORPH]` => `[NOTFOUND_ORPH]false`

Reply:
While you are here rename this ugly `it` param.